README.md

```diff
@@ -160,8 +160,8 @@ administrative privileges):
 
 Now you should be able to create an environment containing Python. Try this:
 
-    ]$ rez-env python -- which python
-    /home/ajohns/packages/python-2.7.8/platform-linux/arch-x86_64/os-Ubuntu-12.04/bin/python
+    ]$ rez-env python -- python -V
+    Python 2.6.8
```

Comment:
The purpose of this is to show that the python executable path has now been standardized within the rez package repository.

Could replace with `$ rez-env python -- python -c "import sys;print(sys.executable)"`, which would be cross-platform